includes/class-coblocks-form.php

```diff
@@ -405,21 +403,24 @@ private function render_submit_button( $atts ) {
 
 		if ( ! empty( $styles ) ) {
 
-			$styles = " style='{$styles}'";
+			$styles = ' style="' . esc_attr( $styles ) . '"';
 
 		}
 
+		// phpcs:disable WordPress.Security.EscapeOutput.OutputNotEscaped
 		?>
 
 		<button type="submit" class="wp-block-button__link <?php echo esc_attr( $btn_class ); ?>"<?php echo $styles; ?>><?php echo esc_html( $btn_text ); ?></button>
```

Comment:
Does the output of `$styles` need to be escaped here?

Reply:
I don’t think it does but Wordpress rules will always trigger the warning. I will solve this a better wat so that we don’t disable rules.

Reply:
Fixed.